pkg/scheduler/framework/runtime/framework.go

```diff
@@ -355,6 +355,15 @@ func NewFramework(ctx context.Context, r Registry, profile *config.KubeScheduler
 		options.captureProfile(outputProfile)
 	}
 
+	m := *f.ListPlugins()
+	pluginMap := reflect.ValueOf(m)
+	typeOfMap := pluginMap.Type()
+
+	for i := 0; i < pluginMap.NumField(); i++ {
+		loggerV := logger.V(2)
+		loggerV.Info("Plugins enabled for", "extension", typeOfMap.Field(i).Name, "plugins", pluginMap.Field(i).Interface())
+	}
+
```

Comment:
IMO just logging a returned value from f.ListPlugins is enough - people should be familiar with the format of `config.Plugins`.

Reply:
Done!!